webapp-client/src/component/ErrorMessage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export default class ErrorMessage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      openWindow: true,
+    }
+    this.displayError = this.displayError.bind(this);
+  }
+
+  displayError() {
+    if (this.props.hasError && this.state.openWindow) {
+      return (
+        <div className="notification is-warning">
+          <button className="delete" onClick={this.props.closePopup}></button>
+          {this.props.message}
+        </div>
+      );
+    }
+    return (
+      <div></div>
+    );
```

Comment:
You can return nothing when not showing this component ([source](https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering)).